src/lib/isIdentityCard.js

```diff
@@ -342,6 +342,37 @@ const validators = {
     };
     return checkIdCardNo(str);
   },
+  'zh-HK': (str) => {
+    // sanitize user input
+    str = str.trim();
```

Comment:
IMHO, this sanitization step does not belong inside the validator, but I guess we can take a look after the release.
related issue #2158

Reply:
Sure.